src/App.js

```diff
@@ -1,57 +1,55 @@
-import React, {Component} from 'react';
+import React, {useEffect} from 'react';
 import {Route, withRouter} from 'react-router-dom';
-import {connect} from 'react-redux';
-import {compose} from 'redux';
+import {useDispatch, useSelector} from 'react-redux';
 
 import Navbar from './components/Navbar/Navbar';
 import News from './components/News/News';
 import Music from './components/Music/Music';
 import Settings from './components/Settings/Settings';
-import DialogsContainer from './components/Dialogs/DialogsContainer';
-import UsersContainer from './components/Users/UsersContainer';
+import {Dialogs} from './components/Dialogs/Dialogs';
+import {UsersContainer} from './components/Users/UsersContainer';
 import ProfileContainer from './components/Profile/ProfileContainer';
-import HeaderContainer from './components/Header/HeaderContainer';
-import LoginPage from './components/Login/Login';
+import {Header} from './components/Header/Header';
+import {Login} from './components/Login/Login';
 import {initializeApp} from './Redux/app-reducer';
 import Preloader from './components/Preloader/Preloader';
+import withAuthRedirect from './auth/withAuthRedirect';
+import {getInitial} from './Selectors/selectors';
```

Comment:
the naming is bad. the file should be just `/selectors/index.js` so you can import it as `import {getInitial} from './selectors';`